lib/sqlalchemy/sql/elements.py

```diff
@@ -411,7 +421,11 @@ def _clone(self: SelfClauseElement, **kw: Any) -> SelfClauseElement:
         c._is_clone_of = cc if cc is not None else self
         return c
 
-    def _negate_in_binary(self, negated_op, original_op):
+    def _negate_in_binary(
+        self: SelfClauseElement,
+        negated_op: Callable[..., Any],
+        original_op: Callable[..., Any],
```

Comment:
Looks like that might be narrowed down to `OperatorType`, will look into that

Reply:
Yes, it's a good idea to do it. I think you can safely narrow it down for any `*_op` parameters.